api/DragEvent.json

```
@@ -116,7 +117,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": "10"
+              "version_added": false
```

This one is tricky. `dataTransfer` is in the prototype for `DragEvent`. But as you've shown, it is always `null`. I'd suggest using a note and `partial_implementation` here instead:

```suggestion
              "version_added": "10",
              "partial_implementation": true,
              "notes": "The value is always <code>null</code>."
```